### PR TITLE
docs: fix double code snippets

### PR DESCRIPTION
**Pull request by mxschmitt**
Fixes https://github.com/microsoft/playwright/pull/20908

### CODE REVIEW

File: docs/src/release-notes-js.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ import LiteYouTube from '@site/src/components/LiteYouTube';
 - Official [docker images](./docker.md) now include Node 18 instead of Node 16.
 
 
-## ⚠️ Breaking change in component tests
+### ⚠️ Breaking change in component tests
```
Comment (mxschmitt) [inline docs/src/release-notes-js.md:72]: drive-by: nesting bug

**Review by debs-obrien — APPROVED** [review]
